sh.fhs.ksre.appdata.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>sh.fhs.ksre</id>
+    <metadata_license>CC-BY-SA-3.0</metadata_license>
+    <name>Katawa Shoujo: Re-Engineered</name>
```

Comment:
can you put it somewhere that this is v2 and not be confused with https://flathub.org/apps/sh.fhs.KatawaShoujoReEngineered? I guess they both have same icons.

Reply:
can I make a change in the old repo to add "Legacy" to its name, and here provide the name intact?

Reply:
No, old repo name or ID cannot be changed. Although you can change its `name` tag, desktop file name etc.

Reply:
I was referring to a name tag and desktop file properties, yep.

Reply:
yes that's fine

Reply:
nice, will do it that way then